client/src/app/panel/tabs/task-testing/__tests__/TaskTestingTabSpec.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import React from 'react';
+
+import { render, waitFor } from '@testing-library/react';
+
+import { SlotFillRoot } from '../../../../slot-fill';
+
+import Panel from '../../../Panel';
+
+import TestTab from '../TaskTestingTab';
+
+import { ENGINES } from '../../../../../util/Engines';
+
+import Modeler from '../../../../../../test/mocks/bpmn-js/Modeler';
+
+
+describe('<TestTab>', function() {
+
+  const WELCOME_MESSAGE = 'Select a task to start testing';
+  const UNSUPPORTED_MESSAGE = 'Task testing is not supported by the current engine version.';
+
+  it('should render', async function() {
+
+    // when
+    const { getByText } = renderTab();
+
+    // then
+    await waitFor(() => {
+      expect(getByText(WELCOME_MESSAGE)).to.exist;
+    });
+
+  });
+
+
+  it('should not support Camunda Cloud < 8.7.0', async function() {
```

Comment:
if we only use the unified rest client, its not supported < 8.8 (|<=8.7)
```suggestion 
  it('should not support Camunda Cloud < 8.8.0', async function() {
```
for support of 8.7 we need the operate client and an operate URL input for it

Reply:
this is affecting some more areas, maybe search for 8.7 in the code to find all occurences